lib/libraries/c3/extensions/paginating-legend.js

```diff
@@ -0,0 +1,217 @@
+var d3 = require('d3');
+
+var isDateString = require('../../../utils/assert-date-string');
+
+module.exports = function(cols){
+  var self = this, chart, columns, domNode, pagination, range;
```

Comment:
Is there any reason we can't define these variables like this?
```js
var self = this,
chart = this.view._artifacts.c3,
columns = []
```

Rather than setting the variables to undefined first, and then defining them? The other files in `keen-dataviz` don't seem to follow this convention, it feels a little inconsistent.